armi/reactor/geometry.py

```diff
@@ -359,8 +359,10 @@ def createValidSymmetryStrings(cls):
     def fromStr(cls, symmetryString: str) -> "SymmetryType":
         """Construct a SymmetryType object from a valid string"""
         canonical = symmetryString.lower().strip()
+        # ignore "assembly" since it is unnecessary and overly-verbose and too specific
+        noAssembly = canonical.replace("assembly", "").strip()
         isThroughCenter = cls._checkIfThroughCenter(canonical)
-        coreString = canonical.replace(THROUGH_CENTER_ASSEMBLY, "").strip()
+        coreString = noAssembly.replace(THROUGH_CENTER_ASSEMBLY, "").strip()
```

Comment:
Is `assembly` going to be in the symmetryString anymore now that the `THROUGH_CENTER_ASSEMBLY ` string below was changed? Also, is there a reason for us keeping the `core` string in symmetryString? This seems a bit fragile long term.

Reply:
they are both optional. `quarter core through center assembly` is the same as `quarter through center`. That' why we are just stripping the words out; they carry no actual meaning

Reply:
Oh interesting. Seems like we should just have one way to define this but as you say below you don't want an API breaking change. Maybe a good issue?